src/providers/Comscore.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Comscore
+ * https://direct.comscore.com/clients/help/FAQ.aspx#faqTagging
+ *
+ * @class
+ * @extends BaseProvider
+ */
+
+class ComscoreProvider extends BaseProvider {
+  constructor() {
+    super();
+    this._key = "COMSCORE";
+    this._pattern = /sb\.scorecardresearch\.com.+(?<!\.js)$/;
+    this._name = "Comscore";
+    this._type = "marketing";
+  }
+
+  /**
+   * Retrieve the column mappings for default columns (account, event type)
+   *
+   * @return {{}}
+   */
+  get columnMapping() {
+    return {
+      account: "c2",
+      requestType: "requestType"
+    };
+  }
+
+  /**
+   * Retrieve the group names & order
+   *
+   * @returns {*[]}
+   */
+  get groups() {
+    return [
+      {
+        key: "general",
+        name: "General"
+      },
+      {
+        key: "custom",
+        name: "Custom"
+      }
+    ];
+  }
+
+  /**
+   * Get all of the available URL parameter keys
+   *
+   * @returns {{}}
+   */
+  get keys() {
+    return {
+      c1: {
+        name: "Tag Type",
+        group: "general"
+      },
+      c2: {
+        name: "Client ID",
+        group: "general"
+      }
+    };
+  }
+
+  /**
+   * Parse a given URL parameter into human-readable form
+   *
+   * @param {string}  name
+   * @param {string}  value
+   *
+   * @returns {void|{}}
+   */
+  handleQueryParam(name, value) {
+    let result = {};
+
+    if (name.startsWith("c")) {
```

Comment:
Consider changing this to regex still, just to match the `c###` format, e.g. `/^c\d+$/`. Also, I'd add an exclusion for c1/c2 since they are "reserved" and not really custom.

Reply:
Good call. Done